hydrator-common/src/main/java/io/cdap/plugin/common/batch/action/ConditionConfig.java

```diff
@@ -44,13 +45,13 @@ public ConditionConfig(String runCondition) {
   }
 
   @SuppressWarnings("ConstantConditions")
-  public void validate() {
+  public void validate(FailureCollector collector) {
```

Comment:
This change should be backwards compatible. We should deprecate this method and add a new method with failure collector.

Reply:
Fixed.